test/extensions.d.ts

```diff
@@ -1,5 +1,16 @@
-declare module Chai {
-	export interface Assertion {
-		equalNode(node: Node | null, message?: string): void;
+declare global {
+	namespace Chai {
+		interface Assertion {
+			equalNode(node: Node | null, message?: string): void;
+			called: Assertion;
+			calledOnce: Assertion;
+			calledTwice: Assertion;
+			calledWith(...args: any[]): Assertion;
+			calledWithMatch(...args: Record<string, unknown>[]): Assertion;
```

Comment:
There's still a few missing, but this should at least cut down on the number of squiggles.